.circleci/code-sniff.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Maybe this could be a composer.json script as well, like build-assets?

https://github.com/drush-ops/drush/blob/9.0.0-beta3/composer.json#L62